src/amo/middleware/responseTime.js

```diff
@@ -13,7 +13,9 @@ export const responseTime = ({
     const client = new _HotShots({
       host: _config.get('datadogHost'),
       port: _config.get('datadogPort'),
-      prefix: 'addons_frontend.server.',
+      // `NODE_CONFIG_ENV` contains the value of `%{::env}` in ops config
+      // ('dev', 'stage' or 'prod').
+      prefix: `addons-frontend-${process.env.NODE_CONFIG_ENV}.server.`,
```

Comment:
if all the events are `.server`, do we need the `.server`?

Reply:
Good question, I think we might not have a way to use statsd on the client but we never know so let's keep it as it is now.